wcfsetup/install/files/lib/system/cache/eager/LanguageCache.class.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace wcf\system\cache\eager;
+
+use wcf\data\DatabaseObject;
+use wcf\data\language\category\LanguageCategoryList;
+use wcf\data\language\LanguageList;
+use wcf\system\cache\eager\data\LanguageCacheData;
+
+/**
+ * Eager cache implementation for languages.
+ *
+ * @author      Olaf Braun
+ * @copyright   2001-2025 WoltLab GmbH
+ * @license     GNU Lesser General Public License <http://opensource.org/licenses/lgpl-license.php>
+ * @since       6.2
+ *
+ * @extends AbstractEagerCache<LanguageCacheData>
+ */
+final class LanguageCache extends AbstractEagerCache
+{
+    #[\Override]
+    protected function getCacheData(): LanguageCacheData
+    {
+        $languageList = new LanguageList();
+        $languageList->getConditionBuilder()->add('language.isDisabled = ?', [0]);
+        $languageList->readObjects();
+
+        $languages = $languageList->getObjects();
+        $default = 0;
+        $multilingualismEnabled = false;
+        $codes = [];
+        $countryCodes = [];
+
+        foreach ($languageList->getObjects() as $language) {
+            // default language
```

Comment:
When copying code, please kick out pointless comments like this one that don’t add any value. In fact, all current comments in this functions are pointless.